js/client-chat.js

```diff
@@ -1022,8 +1036,11 @@
 				}
 			}
 			if (userlist) this.addRow(userlist);
-			if (autoscroll) {
+			if (this.getAutoScroll()) {
 				this.$chatFrame.scrollTop(this.$chat.height());
+			} else if (!this.scrollHandler) {
+				this.scrollHandler = _.bind(ConsoleRoom.scrollHandler, this);
+				this.$chatFrame.on('scroll', this.scrollHandler);
```

Comment:
This is the only scroll handler, and it's intended to clear subtle notifications once the user scrolls down to the bottom of the chat log.